src/PKSim.Core/Snapshots/Mappers/PopulationAnalysisFieldMapper.cs

```diff
@@ -121,8 +121,8 @@ private void mapNumericFieldToSnapshot(PopulationAnalysisField snapshot, INumeri
       private void mapNumericFieldToModel(PopulationAnalysisField snapshot, INumericValueField field)
       {
          field.Dimension = _dimensionFactory.Dimension(snapshot.Dimension);
-         field.Dimension = _dimensionFactory.OptimalDimension(field.Dimension);
```

Comment:
This was the mistake. I was using a merged dimension right away in the domain object. Merged dimension are only for display stuff. So during the construction of the chart, it would not create a merge dim of merged dim and crash lamentably.

I need the optimalDimension only to retrieve a unit instance. Nothing else